### PR TITLE
chore: bump GB

### PR DESCRIPTION
**Pull request by robert-zaremba**
## Description

Updates GB. All claims are enabled, but the slashing is still off.
Partially closes: https://github.com/umee-network/umee/issues/1403
Depends on: https://github.com/umee-network/umee/pull/1478

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [ ] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] added `!` to the type prefix if API or client breaking change
- [ ] added appropriate labels to the PR
- [ ] targeted the correct branch (see [PR Targeting](https://github.com/umee-network/umee/blob/main/CONTRIBUTING.md#pr-targeting))
- [ ] provided a link to the relevant issue or specification
- [ ] added a changelog entry to `CHANGELOG.md`
- [ ] included comments for [documenting Go code](https://blog.golang.org/godoc)
- [ ] updated the relevant documentation or specification
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items._

I have...

- [ ] confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed state machine logic
- [ ] reviewed API design and naming
- [ ] reviewed documentation is accurate
- [ ] reviewed tests and test coverage
- [ ] manually tested (if applicable)

### CODE REVIEW

**Comment by RafilxTenfen** [comment]
why not release umee GB v1.5.3-umee-3 ?

**Comment by toteki** [comment]
"Update SDK" commit has introduced compile time errors @robert-zaremba

**Comment by robert-zaremba** [comment]
> why not release umee GB v1.5.3-umee-3 ?

Because this is not a proper release: slashing is off and messages are on, which should not be marked as a version that anyone should use (except us)

**Comment by robert-zaremba** [comment]
Ping. Kindly requesting review

File: CHANGELOG.md [hunk 1]
```diff
@@ -48,12 +48,15 @@ Ref: https://keepachangelog.com/en/1.0.0/
...
 
+- [1474](https://github.com/umee-network/umee/pull/1474) Enabled all Gravity Bridge claims.
 - [1479](https://github.com/umee-network/umee/pull/1479) Add MsgSupplyCollateral.
 
 ### Fixes
 
 - [1471](https://github.com/umee-network/umee/pull/1471) Fix slash window progress query.
 
+> > > > > > > main
+
```
Comment (toteki) [inline CHANGELOG.md:59]:
```suggestion
```

**Comment by RafilxTenfen** [comment]
> > why not release umee GB v1.5.3-umee-3 ?
> 
> Because this is not a proper release: slashing is off and messages are on, which should not be marked as a version that anyone should use (except us)

Okay, agree with that 

One thing to keep in mind for the next not even a PR was made on https://github.com/umee-network/Gravity-Bridge, only a commit and we referenced that on this go.mod makes it harder to see the changes...
@robert-zaremba

**Pull request merged**